test/unit_tests/services/test_processes.py

```diff
@@ -646,7 +646,7 @@ def run_func():
     assert EngineSettingsTable.query.one().running_processes == 1
 
     event.set()
-    sleep(0.01)
+    sleep(2)
```

Comment:
I'm not a bit fan of having delays in tests. Haven't look into the details, but if there is no we that we can have event.set() return a status, callback, promise, etc. if this is async?

Reply:
```python
>>> h = "hoi"
>>> g = Union[str,int]
>>> x = get_origin(h)
>>> x
>>> x = get_origin(g)
>>> x
typing.Union

```

Reply:
I Added this as it was a flakey test. There are quite a number of calls to .sleep() in this file. I'd rather leave it as is for now instead of refactoring the tests. I agree this could be done better.